pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -166,7 +169,73 @@ let
 
   # Watch out for python <> boost compatibility
   python = python310.override {
```

Comment:
I think we already dropped python 3.10 support, so the situation here will only get much worse over time.

Reply:
Ceph 19 will address this, with Python 3.13 support:

* https://github.com/ceph/ceph/commit/27c05edc8ef6f4b37dc5f0b38f8baff9364117d6
  * https://tracker.ceph.com/issues/62428